reviewpad.yml

```diff
@@ -61,7 +61,7 @@ workflows:
     if:
       - rule: $hasFilePattern("public/data/**")
     then:
-      - '$addLabel("chore: profile-adition")'
+      - '$addLabel("✍ chore: profile-addition")'
```

Comment:
```suggestion
      - '$addLabel("✍ chore: profile addition")'
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix